gamemodes/cinema/gamemode/misc/sh_translations.lua

```diff
@@ -3,40 +3,17 @@ ColDefault = Color(200, 200, 200)
 ColHighlight = Color(158, 37, 33)
 local LANG = {}
 translations = {}
-local Languages = {}
 local DefaultId = "en"
 
-function T(key, ...)
```

Comment:
Can you revert the changes on this file? In case there are any placed you missed. Later I will delete the whole file.

Reply:
Otherwise looks good

Reply:
fixed